html/elements/input/date.json

```diff
@@ -44,6 +44,30 @@
               "standard_track": true,
               "deprecated": false
             }
+          },
+          "disables_invalid_values_in_picker_ui": {
+            "__compat": {
+              "description": "Disables invalid values in picker UI",
+              "support": {
+                "chrome": {
+                  "version_added": "20"
+                },
+                "firefox": {
+                  "version_added": "57"
+                },
+                "safari": {
+                  "version_added": "14.1"
+                },
+                "safari_ios": {
+                  "version_added": false
+                }
```

Comment:
Please note the failing lint, and add all these browsers as `"<browser>": "mirror"`.

> Browser Presence - 1 problem (1 error, 0 warnings):
>  ✖ html.elements.input.type_date.disables_invalid_values_in_picker_ui - Error → Missing the following browsers, which are required for html compat data: chrome_android, edge, firefox_android, opera, opera_android, samsunginternet_android, webview_android, webview_ios